### ISSUE TITLE
Make the Tomsters page easier to find

### ISSUE THREAD

**Issue by acorncom** [emberjs/website#2275]
Right now, in order to get to it, you have to click the Legal link in the footer and then the "view the official Tomsters" link in the middle of the page.

Possible location would be on the Community page (or even nicer, add a tiny Tomster head instead of a map icon on the Meetup page and show it on hover).

Definitely not urgent, so if this is better to close in the interests of focusing on other things, will fully understand :-)

**Comment by wifelette** [comment]
You are indeed correct; I forgot we even put the link in Legal and thought it was even worse. Easter Egg Style, you only get to see it if you already know to go to the direct URL :stuck_out_tongue_winking_eye: 

So I thought about this when I started adding the pages. The main reason I didn't think through a more prominent path was that at that time we were jumping into the optimistic site-wide redesign, and I figured this would be temporary. 

Months later, the redesign isn't coming any time super soon, so we should tackle this sooner. I have some thoughts and will discuss with @sandraor, as she's in the middle of another tweak to the community page. 

I don't think we want to start muddling with the overall hierarchy from this meta angle, but that'll get revisited soon enough. So if we get this on the community page, we should be able to avoid messing with more for now.
